drivers/sx126x/sx126x_netdev.c

```diff
@@ -288,9 +309,17 @@ static int _set_state(sx126x_t *dev, netopt_state_t state)
     case NETOPT_STATE_IDLE:
     case NETOPT_STATE_RX:
         DEBUG("[sx126x] netdev: set NETOPT_STATE_RX state\n");
+        /* Refer Section 4.2 RF Switch in Application Note (AN5406) */
+#if defined(BOARD_NUCLEO_WL55JC)
+        gpio_set(FE_CTRL1);
+        gpio_clear(FE_CTRL2);
+        gpio_set(FE_CTRL3);
+#endif
         sx126x_cfg_rx_boosted(dev, true);
-        if (dev->rx_timeout != 0) {
-            sx126x_set_rx(dev, dev->rx_timeout);
+        /* Get the multiplier for timeout */
```

Comment:
I think the changes related to rx timeout should be part of their own PR.

Reply:
Okay :) Makes sense